api/utils/cron.js

```diff
@@ -0,0 +1,9 @@
+const CronJob = require('cron').CronJob;
+const DatabaseBackup = require('../db/backup.js');
```

Comment:
For consistency with the latest additions to the code, this should be `const { createBackup } = require('../db/backup.js');`